packages/graphql/tests/tck/tck-test-files/subscriptions/update.test.ts

```diff
@@ -73,7 +73,12 @@ describe("Subscriptions metadata on update", () => {
             MATCH (this:Movie)
             WHERE this.id = $this_id
             WITH this { .* } AS oldProps, this, meta
-            SET this.id = $this_update_id
+            CALL {
+            	WITH *
+            	SET this.id = $this_update_id
+            	RETURN meta as update_meta
+            }
+            WITH *, update_meta as meta
```

Comment:
Is there any value in this? Could it not have just been:

```
SET
SET
SET
WITH this, meta + { ...meta }
```

I guess I'm asking, was this possible without the subquery?

Reply:
The problem without subquery is that oldProps will pollute the nested queries, the elegant thing about `CALL { WITH *` is that it completely encapsulates subqueries

Reply:
I have no major objection, just a curiosity thing!